java/fury-core/src/main/java/org/apache/fury/serializer/collection/AbstractMapSerializer.java

```diff
@@ -125,6 +126,32 @@ public void setValueSerializer(Serializer valueSerializer) {
     this.valueSerializer = valueSerializer;
   }
 
+  /**
+   * this method try to eliminate the virtual function call and help JIT inline for java String
+   * type.
+   */
+  private void tryFastSerialize(Serializer serializer, MemoryBuffer buffer, Object value) {
+    if (serializer instanceof StringSerializer) {
```

Comment:
instanceof is slow in java, could we use serializer.getClass is StringSerializer instead?